src/main/java/com/easyrest/components/owner/Details.java

```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Details {
```

Comment:
Is it restaurant details? Maybe named more specifically?

Reply:
![image](https://user-images.githubusercontent.com/104895324/184369504-9ec578fe-15a1-41bb-9e08-a3242b2fcc35.png)

Reply:
All components are in one package, I think there should be no problems with understanding